ArmPkg/Drivers/MmCommunicationDxe/MmCommunication.c

```diff
@@ -259,8 +259,12 @@ GetMmCompatibility (
 
   MmVersion = MmVersionArgs.Arg0;
 
+  STATIC_ASSERT (
+    MM_CALLER_MINOR_VER == 0,
+    "MM_CALLER_MINOR_VERSION has changed!"
+    );
   if ((MM_MAJOR_VER (MmVersion) == MM_CALLER_MAJOR_VER) &&
-      (MM_MINOR_VER (MmVersion) >= MM_CALLER_MINOR_VER))
+      (MM_MINOR_VER (MmVersion) == MM_CALLER_MINOR_VER))

```

Comment:
Makes sense. This can be added to the CodeQL ignore list and dropped from this commit, then. IIRC, the CodeQL error was a useless check, since it is comparing some number >= 0 (the defined minor value) and that is always going to pass.